### PR TITLE
adding redis cluster definition

### PR DESCRIPTION
**Pull request by venturel**

### CODE REVIEW

File: redis/variables.tf [hunk 1]
```diff
@@ -0,0 +1,26 @@
+variable "name" {
```
Comment (MattiasGees) [inline redis/variables.tf:1]: Can you add a description to all the variables?

Comment (venturel) [reply redis/variables.tf:1]: done

File: README.md [hunk 2]
```diff
@@ -1,2 +1,38 @@
...
+ * [`profile_arn`]: String: The ARN assigned by AWS to the instance profile.
+ * [`profile_name`]: String: The instance profile's name.
+ * [`policy_id`]: String: The role policy ID.
+ * [`policy_name`]: String:  The name of the policy.
+ * [`policy_policy`]: String: The policy document attached to the role.
+ * [`policy_role`]: String: The role to which this policy applies.
+
+### Example
+```
+  module "packer_role" {
```
Comment (iuriaranda) [inline README.md]: You've missed to change the example 😉

Comment (venturel) [reply README.md]: shit

**Review by iuriaranda — APPROVED** [review]

**Review by MattiasGees — APPROVED** [review]

**Pull request merged**